www/scipylib/donations.rst

```diff
@@ -57,6 +57,223 @@ infrastructure:
 *This list is ordered by time (most recent contributions first) and was last
 updated in April 2019.*
 
+
+Thanks
+------
+
+SciPy is a community project sponsored by Enthought, Inc.
+SciPy originated with code contributions by Travis Oliphant, Pearu
+Peterson, and Eric Jones.  Travis Oliphant and Eric Jones each contributed
+about half the initial code.  Pearu Peterson developed f2py, which is the
+integral to wrapping the many Fortran libraries used in SciPy.
+
+Below is a partial list of contributors to SciPy.  If you've
+been left off, please email the `SciPy Developers List`_.
```

Comment:
Let's remove this as well, we don't want emails for this.

Reply:
ok